src/components/bottom-tab-nav/tab-details.ts

```diff
@@ -18,23 +18,23 @@ export type TabDetails = {
   icon: string;
 };
 
-export namespace TabInfo {
-  export const SUPPORT: TabDetails = {
+export class TabInfo {
+  public static SUPPORT: TabDetails = {
```

Comment:
Needed to change this (and the one's below) due to this tslint error: `ERROR: no-namespace  'namespace' and 'module' are disallowed`